userguide/system.rst

```diff
@@ -2203,7 +2207,7 @@ information to the report.
 #ifdef truenas
 The %brand% :guilabel:`Support` tab, shown in
 :numref:`Figure %s <tn_support1>`,
-is used to view or update the system's license information. It also
+is used to view or update the license information of the system. It also
```

Comment:
Don't work so hard:

is used to view or update the system license information.